### PR TITLE
IA-582 login language selector

### PR DESCRIPTION
**Pull request by beygorghor**
Screenshot 2021-07-06 at 14 27 05
Screenshot 2021-07-06 at 14 27 11

### CODE REVIEW

**Comment by beygorghor** [comment]
Also re-styling a bit links and buttons.

**Comment by quang-le** [comment]
On big screens, the picker is a bit isolated, I didn't see it at first:

Screenshot 2021-07-06 at 14 43 49

**Comment by quang-le** [comment]
The selected language doesn't carry over to Iaso, as it saved in the `django_language` cookie, and the react app uses the `iaso_locale` value from local storage

https://user-images.githubusercontent.com/38907762/124602554-44ac3180-de69-11eb-8749-8fb2b35bbce8.mov

**Comment by beygorghor** [comment]
@quang-le , Picker is moved and we are now using the django cookie instead of locale storage

File: hat/assets/js/apps/Iaso/utils/cookies.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        let c = ca[i];
+        while (c.charAt(0) === ' ') {
+            c = c.substring(1, c.length);
+        }
+        if (c.indexOf(nameEQ) === 0) {
+            return c.substring(nameEQ.length, c.length);
+        }
+    }
+
+    return null;
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/utils/cookies.js:29]: Suggestion to replace the body of `getCookie`:
```javascript
  const nameEQ = `${name}=`;
    const cookiesArray = document.cookie
        .split(';')
        .map(cookieString => cookieString.trim())
        .filter(cookie => cookie.indexOf(nameEQ) === 0);

    if (cookiesArray.length === 0) {
        return null;
    }
    const cookie = cookiesArray[0];
    return cookie.substring(nameEQ.length, cookie.length);
```

**Review by quang-le — APPROVED** [review]
I think the changes I suggested make the code a bit more readable, but the feature already works well as is

**Review by olethanh — APPROVED** [review]

**Pull request merged**